ovs/extensions/services/upstart.py

```diff
@@ -91,18 +92,27 @@ def add_service(name, client, params=None, target_name=None, additional_dependen
             client.file_write('/etc/init/{0}.conf'.format(target_name), template_file)
 
     @staticmethod
-    def get_service_status(name, client):
+    def get_service_status(name, client, return_output=False):
```

Comment:
I don't care too much what people pass into the ```return_output``` argument, if the caller wants output, he should pass in ```True```. If they want to pass in their grandma that's fine for me, but it won't give them output.